contracts/abi/EIP20Token.abi

```diff
@@ -0,0 +1,240 @@
+        "inputs": [],
+        "name": "symbol",
+        "outputs": [
+            {
+                "name": "tokenSymbol_",
+                "type": "string"
+            }
+        ],
+        "payable": false,
+        "stateMutability": "view",
+        "type": "function"
+    },
+    {
+        "constant": false,
+        "inputs": [
+            {
+                "name": "_to",
+                "type": "address"
+            },
+            {
+                "name": "_value",
+                "type": "uint256"
+            }
+        ],
+        "name": "transfer",
+        "outputs": [
+            {
+                "name": "success_",
+                "type": "bool"
+            }
+        ],
+        "payable": false,
+        "stateMutability": "nonpayable",
+        "type": "function"
+    },
+    {
+        "constant": true,
+        "inputs": [
+            {
+                "name": "_owner",
+                "type": "address"
+            },
+            {
+                "name": "_spender",
+                "type": "address"
+            }
+        ],
+        "name": "allowance",
+        "outputs": [
+            {
+                "name": "allowance_",
+                "type": "uint256"
+            }
+        ],
+        "payable": false,
+        "stateMutability": "view",
+        "type": "function"
+    },
+    {
+        "inputs": [
+            {
+                "name": "_symbol",
+                "type": "string"
+            },
+            {
+                "name": "_name",
+                "type": "string"
+            },
+            {
+                "name": "_totalSupply",
+                "type": "uint256"
+            },
+            {
+                "name": "_decimals",
+                "type": "uint8"
+            }
+        ],
+        "payable": false,
+        "stateMutability": "nonpayable",
+        "type": "constructor"
+    },
+    {
+        "anonymous": false,
+        "inputs": [
+            {
+                "indexed": true,
+                "name": "_from",
+                "type": "address"
+            },
+            {
+                "indexed": true,
+                "name": "_to",
+                "type": "address"
+            },
+            {
+                "indexed": false,
+                "name": "_value",
+                "type": "uint256"
+            }
+        ],
+        "name": "Transfer",
+        "type": "event"
+    },
+    {
+        "anonymous": false,
+        "inputs": [
+            {
+                "indexed": true,
+                "name": "_owner",
+                "type": "address"
+            },
+            {
+                "indexed": true,
+                "name": "_spender",
+                "type": "address"
+            },
+            {
+                "indexed": false,
+                "name": "_value",
+                "type": "uint256"
+            }
+        ],
+        "name": "Approval",
+        "type": "event"
+    }
+]
```

Comment:
Good question. So far we were of the understanding that providing EIP20 code should not be the responsibility of mosaic, as that is a step the user usually takes *before* interacting with mosaic. However, if the user only needs a standard EIP20 token, it would be *very* convenient to provide it from the mosaic layer.

/cc @deepesh-kn @sarvesh-ost @benjaminbollen

Reply:
More than Mosaic, BrandedToken is concerned with EIP20 tokens.  I dont feel EIP20 abi is misplaced here; rather my concern is that we want to move Abi (in this case no bin) out of the interaction layer into a `-contracts` published npm package.

currently BrandedToken.js depends on `mosaic.js`, but not on `mosaic-contracts.js`; therefor I *think* we should get this Abi from BrandedToken-contracts' `AbiBinProvider`

Reply:
Valid point 👍